core/usage_interceptor.go

```diff
@@ -77,7 +78,7 @@ func (t *Textile) preUsageFunc(ctx context.Context, method string) (context.Cont
 	now := time.Now()
 
 	// Collect new users.
-	if account.User.CreatedAt.IsZero() && account.User.Type == mdb.User {
+	if account.User != nil && account.User.CreatedAt.IsZero() && account.User.Type == mdb.User {
```

Comment:
Primary cause of the NPDs seen in production. This commit has already been deployed. No more restarts.